dir.targets

```diff
@@ -56,4 +56,11 @@
     </SupplementalTestData>
   </ItemGroup>
 
+  <Target Name="ProducesPackageId" 
+          Returns="@(PackageIds)">
+    <ItemGroup>
+      <PackageIds Include="$(Id)" />
```

Comment:
This seems too broad, any project that happens to define an Id property will set this.